packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -19,6 +19,7 @@ import { ENTER, BACKSPACE, DELETE } from '@wordpress/keycodes';
 import { __, sprintf } from '@wordpress/i18n';
 import { useSelect, useDispatch } from '@wordpress/data';
 import deprecated from '@wordpress/deprecated';
+import { getBlockProps } from '@wordpress/blocks';
```

Comment:
do you think we should keep this as `__unstable` (`__internal` is probably better but that should be a separate discussion as it's not limited to this API)

Reply:
Sounds good